elm-app/src/elm/Events/Test.elm

```diff
@@ -239,6 +251,26 @@ event3 =
       , description = Just "Evening event description"
       , date = Date.fromTime 0
       , endDate = Just (Date.fromTime 120000000000)
+      , recurringWeekly = False
```

Comment:
Just a note - out of scope for this PR, and probably for after we get real feedback - but I could imagine some events are recurring once every two weeks/ month/ 3 months, etc'.

So while now we have `recurringWeekly` I imagine it becoming `recurring  : Recurring` where

```elm
type Recurring 
  =  NonReuccring
  | ReccuringDay Int -- `Int` indicates how many days
  | ReccuringWeek Int
  | ReccuringMonth Int
```

Reply:
@noamoss Worth keeping this in mind ^^ (if there's other than weekly recurring events), this will need a structrual change in the backend as well.

Reply:
👍 Actually, this was something we originally had in mind, wanted to add and gave up under the 'keep it basic' discipline.

Reply:
> 'keep it basic' discipline

That was a good call 👍